app.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
```

Comment:
Really great constructor function, I can tell you already understand parameters when dealing with functions.